regression-tests/test_LogLevel.py

```diff
@@ -0,0 +1,28 @@
+import subprocess
+import time
+from test_helper import ApiTestCase
+
+class TestBasics(ApiTestCase):
+
+    def check_loglevel(self, loglevel, content):
+        cmd = ("../wforce/wforce -D -C ./wforce3.conf -R ../wforce/regexes.yaml --loglevel %d" % loglevel).split()
+        proc = subprocess.Popen(cmd, stdout=subprocess.PIPE)
+        time.sleep(1)
+        proc.terminate()
+        proc.wait()
+
+        out, err = proc.communicate()
+        if out:
+            out = out.decode()
+        if err:
+            err = err.decode()
+        assert content in out, out
+
+    def test_debug(self):
+        self.check_loglevel(7, "TRACE")
+
+    def test_info(self):
+        self.check_loglevel(5, "ACL allowing queries from")
+
+    def test_warning(self):
+        self.check_loglevel(4, "Warning - webserver() configuration")
```

Comment:
This is probably not a good check - at some point the config will move to using the non-deprecated addListener() config, at which point this log line will disappear and this test will randomly fail.